graphql-server/go-server/graph/datasource.resolvers.go

```diff
@@ -0,0 +1,49 @@
+package graph
```

Comment:
Need a copyright?

Reply:
The regeneration will clean up the license, so I think it can be done without it.